src/Service/KeyGenerator.php

```diff
@@ -0,0 +1,23 @@
+<?php
+declare(strict_types=1);
+
+namespace MyOnlineStore\EventSourcing\Service;
```

Comment:
As you already have various services spread across (more specific) namespaces, I would continue that naming pattern and move this to an `encryption` (or similar) namespace